open_seq2seq/encoders/tacotron2_encoder.py

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+from tensorflow.contrib.cudnn_rnn.python.ops import cudnn_rnn_ops
+from tensorflow.contrib.rnn import LSTMStateTuple
+from tensorflow.python.framework import ops
+
+from .encoder import Encoder
+
+def conv1d_bn_actv(name, inputs, filters, kernel_size, activation_fn, strides,
```

Comment:
Could you reuse the function xravitejax made for general conv_bn_actv?

Reply:
I have changed to the general conv_bn_actv function